osmomath/decimal.go

```diff
@@ -494,6 +496,27 @@ func (d BigDec) MustFloat64() float64 {
 	}
 }
 
+// SdkDec returns the Sdk.Dec representation of a BigDec.
+// Values in any additional decimal places are truncated.
+func (d BigDec) SdkDec() sdk.Dec {
```

Comment:
nit: Would ensure all `sdk` usage in methods and functions is named `SDK`

Reply:
Done!